WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -198,4 +198,31 @@ - (BOOL)fb_openUrl:(NSString *)url error:(NSError **)error
   return YES;
 }
 
+- (BOOL)fb_pressButton:(NSString *)buttonName error:(NSError **)error
+{
+  NSMutableArray<NSString *> *supportedButtonNames = [NSMutableArray array];
+  XCUIDeviceButton dstButton = 0;
+  if ([buttonName.lowercaseString isEqualToString:@"home"]) {
+    dstButton = XCUIDeviceButtonHome;
+  }
+  [supportedButtonNames addObject:@"home"];
+#if !TARGET_OS_SIMULATOR
```

Comment:
am I reading this right to say that volume up and down only work on real devices?